core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -521,14 +521,14 @@ class AutoTuner(
    *         or "spark.executor.memoryOverheadFactor".
    */
   def memoryOverheadLabel: String = {
-    val sparkMasterConf = getPropertyValue("spark.master")
```

Comment:
In the user tools, seems like 'master' is mapped to 'driver'. Is this a special case here?

Reply:
Refactored to driver